### DISCUSSION TITLE
Array properties in config

### DISCUSSION THREAD

**Discussion by KirillTregubov** [tailwindlabs/tailwindcss#12064] [Help] [answered]
I was trying to use the new `dvh` unit but wanted to include a fallback in case it wasn't supported in a browser and came across [this StackOverflow thread](https://stackoverflow.com/questions/75079019/tailwind-css-fallback-for-new-screen-length-types-such-as-lvh-svh) that suggested I add the following to my config:

```js
theme: {
  extend: {
    height: {
      'dynamic-screen': ['100vh /* fallback */', '100dvh']
    }
  }
}
```

which worked and generated the correct CSS:
```css
.h-dynamic-screen {
    height: 100vh /* fallback */;
    height: 100dvh;
}
```

However since I am using typescript for my config, I am getting a type error that "Type 'string[]' is not assignable to type 'string'".

I could not find anything about this in the documentation (I apologize if I missed it), but since this is working it would be beneficial to update the types to reflect and support this behaviour.

**Comment by wongjn** [comment]
I believe the array syntax works by coincidence and is not officially supported, hence why you get the typing error and why there is no documentation.

**Comment by alex-krasikau** [comment]
Wouldn't it be great to add simple support for fallbacks? Or document this approach if it's implemented and works as expected?

**Comment by rozsazoltan** [comment, marked as answer]
It may already be outdated, but I came across the question you mentioned on StackOverflow and collected my thoughts. I won't duplicate it here, so I'll just leave the link: [Tailwind CSS fallback for new screen length types such as "lvh", "svh"](https://stackoverflow.com/a/78998325/15167500)
